frontend/src/libs/components/meditation-timer/meditation-timer.tsx

```diff
@@ -0,0 +1,65 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { useCallback, useState } from '#libs/hooks/hooks.js';
+
+import { Button } from '../components.js';
+import { TimerButton } from './components/duration-button/duration-button.js';
+import { DURATION_UNIT, MEDITATION_DURATION } from './libs/constants.js';
+import styles from './styles.module.scss';
+
+type TimerProperties = {
+  onClose: () => void;
+  defaultDuration: string;
+};
+
+const MeditationTimer: React.FC<TimerProperties> = ({
+  onClose,
+  defaultDuration,
+}) => {
+  const [selectedDuration, setSelectedDuration] = useState(defaultDuration);
+
+  const handleClick = useCallback((duration: string) => {
+    return () => {
+      setSelectedDuration(duration);
+    };
+  }, []);
+
+  return (
+    <div className={styles['timer']}>
+      <p className={styles['title']}>Choose Your Duration</p>
+      <div className={styles['duration-container']}>
+        {Object.keys(MEDITATION_DURATION).map((duration) => {
+          return (
+            <TimerButton
+              key={duration}
+              isActive={selectedDuration === duration}
+              onClick={handleClick(duration)}
+            >
+              <div
```

Comment:
Let's move it to TimerButton indeed

Reply:
Hi, Mister @v0ldemar01 are we talking about moving the `'button-content'` div inside the TimerButton? Initially we [discussed](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/253#issuecomment-1717446896) that it should be a common button that excepts children

<img width="829" alt="Screenshot 2023-09-14 at 11 58 37" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/72ec9044-0b7d-44b6-a04b-e4fd0dbcfc9e">

Reply:
Well, yes, my bad, there is a component that I build for this purpose. 
Let me make sure we are on the same page: 
We need to move the `'button-content'` div inside of the TimerButton component? 
So TimerButton should not accept children anymore?

Reply:
No children, but via props

Reply:
You implement and we will see whether we are on same page